x-pack/legacy/plugins/painless_lab/public/components/output_pane/output_pane.tsx

```diff
@@ -59,7 +57,8 @@ export function OutputPane({
         tabs={[
           {
             id: 'output',
-            name: outputTabLabel,
+            // TODO: Currently this causes an Eui prop error because it is expecting string, but we give it React.ReactNode - should fix.
+            name: outputTabLabel as any,
```

Comment:
This should be a pretty straightforward fix in EUI. The underlying `EuiTab` component already [accepts a node](https://github.com/elastic/eui/blob/master/src/components/tabs/tab.tsx#L58), so it should be a one-liner fix to change `EuiTabbedContent` to [accept a node too](https://github.com/elastic/eui/blob/master/src/components/tabs/tabbed_content/tabbed_content.tsx#L18). There may be Jest snapshots that also require updating.